src/view/com/modals/ChangeHandle.tsx

```diff
@@ -106,13 +99,22 @@ export function Component({onChanged}: {onChanged: () => void}) {
     )
   }, [setCustom, isCustom, track])
   const onPressSave = React.useCallback(async () => {
-    setError('')
-    setProcessing(true)
+    if (!userDomain) {
+      logger.error(`ChangeHandle: userDomain is undefined`, {
+        service: serviceInfo,
+      })
+      setError(`The service you've selected has no domains configured.`)
+      return
```

Comment:
Not sure this will ever happen, but if a PDS returns no domains at least we'll get a log in Sentry and alert the user

Reply:
Yep report an error